cmd/createtree/main.go

```diff
@@ -76,7 +77,12 @@ func createTree(ctx context.Context) (*trillian.Tree, error) {
 		return nil, err
 	}
 
-	conn, err := grpc.Dial(*adminServerAddr, grpc.WithInsecure())
+	dialOpts, err := rpcflags.NewClientDialOptionsFromFlags()
+	if err != nil {
+		return nil, err
```

Comment:
Maybe decorate the error here like you do in `deletetree/main.go` for consistency?

Reply:
Done.